src/Element/Element.php

```diff
@@ -82,16 +82,6 @@ protected function getDriver()
     }
 
     /**
-     * Returns selectors handler.
-     *
-     * @return SelectorsHandler
-     */
-    protected function getSelectorsHandler()
```

Comment:
@aik099 you could use ``method_exists(NamedSelector::class, 'escapeLocator')`` to detect that this PR is included in the codebase if you want to keep the Mink 1.6 compatibility.

And when building your own Xpath, you should indeed use the Escaper class (which is available in 1.6.0)